scripts/BuildMultitoolForNpm.ps1

```diff
@@ -0,0 +1,75 @@
+<#
+.SYNOPSIS
+    Build, and package the Sarif Multitool NPM package.
```

Comment:
>Sarif Multitool [](start = 27, length = 15)

Sarif.Multitool (or "SARIF Multitool" if the idea is to describe it rather than name it).

Reply:
Fixed